conf/machine/poplar.conf

```diff
@@ -0,0 +1,41 @@
+#@TYPE: Machine
+#@NAME: ToCoding Poplar machine
+#@DESCRIPTION: ToCoding Poplar machine
+
+require conf/machine/include/arm/arch-armv8.inc
+
+PREFERRED_PROVIDER_virtual/xserver ?= "xserver-xorg"
+
+XSERVER ?= "xserver-xorg \
+            mesa-driver-swrast \
+            xf86-input-evdev \
+            xf86-video-fbdev \
+           "
+
+MACHINE_FEATURES = "usbhost usbgadget alsa screen ext2"
+
+PREFERRED_PROVIDER_virtual/kernel ?= "linux-poplar"
+
+KERNEL_IMAGETYPE = "Image"
+KERNEL_DEVICETREE = "hisilicon/hi3798cv200-poplar.dtb"
+
+UBOOT_MACHINE = "poplar_defconfig"
+
+
+SERIAL_CONSOLES = "115200;ttyAMA3 115200;ttyFIQ0"
```

Comment:
ttyFIQ0? are we using an AOSP kernel? otherwise, it's another copy/paste from HiKey.